docs/cookbook/recipe_workflow_integration.rst

```diff
@@ -97,5 +92,3 @@ you will see something like this:
 
 
 .. _`Workflow Component`: https://symfony.com/doc/current/components/workflow.html
-.. _`Packagist`: https://packagist.org/packages/yokai/sonata-workflow
-.. _`GitHub`: https://github.com/yokai-php/sonata-workflow
```

Comment:
I would leave this reference somewhere, so people can check the package docs

Reply:
I am against this, otherwise we need to add it everywhere where we install a lib, and how to do this should be a requirement...